Portfolio/src/views/HomeView.vue

```diff
@@ -1,134 +1,144 @@
 <script setup lang="ts">
+import Wrapper from "@/Wrapper.vue";
 import Copyright from "../components/Copyright.vue"
 import { homeViewContent } from "./home-view-content";
 
 
-
 </script>
 
 <template>
-    <!-- default layout -->
-    <header class="home-wrapper">
-        <div class="homepage-leftcontainer">
-            <h2 class="home-smallName">{{ homeViewContent.firstName }}</h2>
-            <h1 class="homepage-title">{{ homeViewContent.titleCareer }}</h1>
-        </div>
-        <div class="homepage-rightcontainer">
-            <h1 class="home-firstName">{{ homeViewContent.firstName }}</h1>
-            <h1 class="home-lastName">{{ homeViewContent.lastName }}</h1>
-        </div>
-    </header>
-    <!-- default layout -->
+    <Wrapper>
+
+        <!-- default layout -->
+        <header class="home-wrapper">
```

Comment:
Is it a home-wrapper or a home-page-wrapper?

And u already have a wrapper component above, so it is confusing why do u need a second wrapper here!